wazo_calld/plugins/applications/events.py

```diff
@@ -154,3 +154,8 @@ class SnoopDeleted(_BaseSnoopItemEvent):
 class SnoopUpdated(_BaseSnoopItemEvent):
     name = 'application_snoop_updated'
     routing_key = 'applications.{}.snoops.{}.updated'
+
+
+class UserOutgoingCallCreated(_BaseCallItemEvent):
+    name = 'application_user_outgoing_call_created'
+    routing_key = 'applications.{}.useroutgoingcall.{}.created'
```

Comment:
I will not be shocked if we use underscore in routing key

Reply:
fixed